### ISSUE TITLE
Problems with dumping the plants from Conker's Bad Fur Day

### ISSUE THREAD

**Issue by Philgamerk98** [gonetz/GLideN64#2869]
Strangely when I dump the plants with this, they look distorted or missing parts of the texture? 
![CONKER BFD#3D9D5DAD#0#2_all](https://github.com/user-attachments/assets/ae43816d-511d-4286-a40a-6d4eed6dade9)
![CONKER BFD#45D564ED#0#2_all](https://github.com/user-attachments/assets/67ed41a7-7f30-49ac-badd-6ff5ad610093)

I don't have this issue when I dump the plants with Lemmy Plugin though. Im using Gliden64 X86 QT exactly following the instructions carefully and this was my first time dumping textures with this tool but they look missed up? strangely this is not a issue with Lemmy and only Gliden64. 

Heres more

![CONKER BFD#00289482#0#2_all](https://github.com/user-attachments/assets/9e66dce9-01c9-4350-9fa3-fa213a0fc13d)
![CONKER BFD#5229AB89#0#2_all](https://github.com/user-attachments/assets/0c7c0497-6ed4-4c79-86fe-e222cbcc3ff4)

**Comment by Philgamerk98** [comment]
bad unreliable plugin, very inactive and unsupportive repository maker here. 👎

**Comment by Xii-Nyth** [comment]
should not be an issue, does it display improperly when using that dump as the replaced texture? 
P.S. many games have junk like this on at least a few textures and its typically on the games side, not the plugin

**Comment by Philgamerk98** [comment]
> should not be an issue, does it display improperly when using that dump as the replaced texture? P.S. many games have junk like this on at least a few textures and its typically on the games side, not the plugin

finally a reply! thank goodness! You should have been here earlier before I closed it. Anyway, I think I know what the problem might be. I was looking at the alpha channel textures, they are only seen when you rip with Lemmy, not this plugin which doesn't include it or possibly hiding them. The alpha textures are black or white depends on the texture, with a white or black shape of the pattern from the texture. 

The game runs fine with no issues on the plugin. In fact, the plugin made the picture look better than Lemmy or Jabo! Only its a bit of a pain to set it up if you know which file is the plugin and which folder is right to put it. The repository didn't even say or at least it should recommend the QT version for 86x programs (my pc is a 64 bit and I was confused when I saw the 64 bit and 86 bit but didnt work when I installed the 64 bit version and the 86 work for me because its installed on that folder due to the program Im using doesn't seem to have a 64 bit version). So theres some improvement that needs to be done here with the description plus making a tutorial for their users would be more helpful if only Github supports posting video instead of uploading it to Youtube. 

 About my issue with the GlideN64, I believe this might have something to do with the alpha channel and not the game itself. The alpha textures I got from Lemmy matches the look of the missed up textures I got from GlideN64! So I figured this must be the problem. Lemmy rips better because the alpha textures are separated but I don't really want those! Is there a way to disable dumping the textures with the alpha ones? So that way, I can only get the real textures without the stupid alpha on them. I think this might help if we don't include the alpha textures when we start dumping the real ones. 

![problem 3](https://github.com/user-attachments/assets/18c761c8-c7f0-4ece-8872-b0cd6dc15e67)
![problem 2](https://github.com/user-attachments/assets/2b7ec8bc-ccb0-4fcd-92ac-91523c18b768)
![problem](https://github.com/user-attachments/assets/c39abea9-68f4-415b-b8d5-a6c105816999)

**Comment by Xii-Nyth** [comment]
Yes the 32 bit/64 bit versions are to match if the emulator. Old ones like Project 64 and 1964 are 32 bit while mupen's modern derivatives are all 64 bit. Unfortunately some people keep editing out critical information and only leaving what they think matters on the various wikis. I had at one point tested all the emulators for windows and listed their capabilities...

Is lemmy a video plugin or an external tool? Ive never heard of it before?

As for disabling alpha textures, there is no specific option, but you can prevent them from repeating dumping by using them as a replacement. GLide wont dump a texture it is already actively replacing. Also, alpha textures tend to be a very low resolution, in windows, if you open the "group by" context menu, you can add more options. If you add dimensions, you can isolate all the low res textures and toggle viewing them on and off, if you are trying to sort through the junk.

**Comment by Philgamerk98** [comment]
Lemmy come from Nemu 64, another emulator but it works better with Project 64 combined but still not as good like GlideN64. My only issue is trying to disable dumping the alpha channel so I only get the textures I wanted and not the alpha because it ruins the textures I believe!
